pkgs/applications/altcoins/zcash/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, libsodium, fetchFromGitHub, wget, pkgconfig, autoreconfHook, openssl, db62, boost
+, zlib, gtest, gmock, miniupnpc, callPackage, gmp, qt4, utillinux, protobuf, qrencode, libevent
+, withGui }:
+
+let snark = callPackage ./snark { inherit boost openssl; };
+in
+with stdenv.lib;
+stdenv.mkDerivation rec{
+
+  name = "zcash" + (toString (optional (!withGui) "d")) + "-" + version;
+  version = "1.0.7";
+
+  src = fetchFromGitHub {
+    owner = "zcash";
+    repo  = "zcash";
+    rev = "d9bfce7539fe5d25c75dae4a0dfebce2e519105f";
+    sha256 = "1axv15ssw8rzifddwjh2axgb37f1cmdxhs0wprv6q5v51hc8xrzw";
```

Comment:
Confirmed; imma bump the version to 1.0.8 to double as to fix with the hash inconsistency.